build.gradle

```diff
@@ -12,7 +12,7 @@ subprojects {
       mavenCentral()
     }
     dependencies {
-      classpath 'com.android.tools.build:gradle:1.2.2'
+      classpath 'com.android.tools.build:gradle:1.2.3'
```

Comment:
should we do something to the line below that has `androidPlugin = 'com.android.tools.build:gradle:1.2.0'` ?   Not even sure what that's used for.